ddtrace/llmobs/_constants.py

```diff
@@ -50,9 +50,9 @@
 # Used to differentiate traces of Datadog-run operations vs user-application operations.
 RUNNER_IS_INTEGRATION_SPAN_TAG = "runner.integration"
 
-# The ml app of all ragas traces have this prefix that we use to detect
-# whether a span is generated from the ragas evaluation itself.
-RAGAS_ML_APP_PREFIX = "dd-ragas"
+# All ragas traces have this context item set so we can differentiate
+# spans generated from the ragas integration vs user application spans.
+IS_EVALUATION_SPAN = "_is_evaluation_span"
```

Comment:
```suggestion
IS_EVALUATION_SPAN = "_ml_obs.evaluation_span"
```
Let's use the _ml_obs prefix to make it clear this is coming from us (for future reviewers/users)